README.md

```diff
@@ -42,19 +42,20 @@ if valid, err := validator.ValidateStruct(reqData); !valid || err {
 "language"              : IsLanguage
 "topic"                 : IsTopic
 "displayName"           : IsDisplayName
-"userDisplayName"       : IsUserDisplayName
+"personName"            : IsPersonName
 "uuid4WithoutHyphens"   : IsUUID4WithoutHyphens
-"orderNumber"           : IsOrderNumber
-"dockerImage"           : IsDockerImage
 "permissionResource"    : IsPermissionResource
 "path"                  : IsPath
 "url"                   : IsURL
-"memorySize"            : IsMemorySize
-"time"                  : IsTime
-"dob"                   : IsDate
+"dateTime"              : IsDateTime
+"date"                  : IsDate
 "jwt"                   : IsJWT
 "password"              : IsPassword
-"emailOWASP"            : IsOWASPEmail
+"email"                 : IsOWASPEmail
```

Comment:
why do we need OWASP in the name?

Reply:
did you mean the name of the function?
after thinking about it, we don't need the OWASP word in the function name.
we only need it in the pattern to tells where did we get those pattern.

i'll update it